help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,135 @@
+# Move from self-hosting to Zulip Cloud
+
+With Zulip's high quality import and export tools, you can always
+move from self-hosting your own Zulip server to using the Zulip
+Cloud service (and back).
+
+## Process overview
+
+To move your Zulip organization from a self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Plan the process and coordinate with Zulip support](#plan-the-process-and-coordinate-with-zulip-support).
+
+1. [Upgrade your self-hosted server](#upgrade-your-self-hosted-server).
+
+1. [Export organization data](#export-organization-data).
+
+1. [Decide how users will log in](#decide-how-users-will-log-in).
+
+{end_tabs}
+
+## Plan the process and coordinate with Zulip support
+
+To import your self-hosted organization into Zulip Cloud, your server will need
+to have the same database version as Zulip Cloud. Zulip Cloud is updated every
+couple of weeks, so it's important to coordinate the timing with Zulip's support
+team.
+
+{start_tabs}
+
+1. Review the process described on this page, and decide when you will be ready
+   to make the transition.
+
+1. [Create a new Zulip Cloud organization](https://zulip.com/new/)
+   with the subdomain you would like for your Zulip organization. Your
+   Zulip Cloud organization will be hosted at `<subdomain>.zulipchat.com`.
```

Comment:
maybe this should say "to reserve the subdomain you would like ..."? Could try to be even more clear that the organization won't be used.

This step isn't required at all; I would generally advise users to skip it. 

Also, `templates/zerver/create_realm.html` should link to this page as it does for other popular data import tools.

Reply:
If they don't need to, then we should just tell them to tell us what domain they want.

Reply:
> Also, templates/zerver/create_realm.html should link to this page as it does for other popular data import tools.

I assume someone else is taking care of this.